src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1000,11 +1000,17 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
-        [ActiveIssue(22187, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [Theory, MemberData(nameof(HeaderWithEmptyValueAndUris))]
         public async Task GetAsync_RequestHeadersAddCustomHeaders_HeaderAndEmptyValueSent(string name, string value, Uri uri)
         {
+            if (PlatformDetection.IsWindows && !PlatformDetection.IsWindows10Version1709OrGreater)
```

Comment:
Why not use [ConditionalTheory] instead?

Reply:
We've used this pattern in many places already in this file.  We do have a few other places though where we use CondtionalTheory. I can switch to that if you want.

Thinking about this particular test a little more, we only want it to be skipped during UAP runs and only if it is NOT "Fall Creators Update" or newer.  But in non-UAP cases, we can use whatever Windows build.  So, I can update that also.